vips/vip-498/bscmainnet.ts

```diff
@@ -0,0 +1,347 @@
+export const USDT = "0x55d398326f99059fF775485246999027B3197955";
+
+const STALE_PERIOD_26H = 26 * 60 * 60; // heartbeat of 24H
+export const CONVERSION_INCENTIVE = parseUnits("1", 14);
+
+export const convertAmountToVTokens = (amount: BigNumber, exchangeRate: BigNumber) => {
+  const EXP_SCALE = parseUnits("1", 18);
+  return amount.mul(EXP_SCALE).div(exchangeRate);
+};
+
+type Token = {
+  address: string;
+  decimals: number;
+  symbol: string;
+};
+
+export const weETH: Token = {
+  address: "0x7DCC39B4d1C53CB31e1aBc0e358b43987FEF80f7",
+  decimals: 18,
+  symbol: "weETH",
+};
+
+export const wstETH: Token = {
+  address: "0xc02fE7317D4eb8753a02c35fe019786854A92001",
+  decimals: 18,
+  symbol: "wstETH",
+};
+
+type Market = {
+  vToken: {
+    address: string;
+    name: string;
+    symbol: string;
+    underlying: Token;
+    decimals: number;
+    exchangeRate: BigNumber;
+    comptroller: string;
+  };
+  riskParameters: {
+    collateralFactor: BigNumber;
+    liquidationThreshold: BigNumber;
+    supplyCap: BigNumber;
+    borrowCap: BigNumber;
+    reserveFactor: BigNumber;
+    protocolSeizeShare: BigNumber;
+  };
+  initialSupply: {
+    amount: BigNumber;
+    vTokensToBurn: BigNumber;
+    vTokenReceiver: string;
+  };
+  interestRateModel: {
+    address: string;
+    base: string;
+    multiplier: string;
+    jump: string;
+    kink: string;
+  };
+};
+
+export const weETHMarket: Market = {
+  vToken: {
+    address: "0xd16b6f7f5b51e402fC1190dc6fc990a2a6ED794f",
+    name: "Venus weETH (Core)",
+    symbol: "vweETH_Core",
+    underlying: weETH,
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+  riskParameters: {
+    collateralFactor: parseUnits("0.7", 18),
+    liquidationThreshold: parseUnits("0.75", 18),
+    supplyCap: parseUnits("4000", 18),
+    borrowCap: parseUnits("400", 18),
+    reserveFactor: parseUnits("0.4", 18),
+    protocolSeizeShare: parseUnits("0.05", 18),
+  },
+  initialSupply: {
+    amount: parseUnits("3", 18),
+    vTokensToBurn: parseUnits("0.03694", 8), // around $100
+    vTokenReceiver: unichainmainnet.VTREASURY,
+  },
+  interestRateModel: {
+    address: "0xc2edB1e96e45178b901618B96142C2743Ed0e7B3",
+    base: "0",
+    multiplier: "0.09",
+    jump: "3",
+    kink: "0.45",
+  },
+};
+
+export const wstETHMarket: Market = {
+  vToken: {
+    address: "0x3B3aCc90D848981E69052FD461123EA19dca6cAF",
+    name: "Venus wstETH (Core)",
+    symbol: "vwstETH_Core",
+    underlying: wstETH,
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+  riskParameters: {
+    collateralFactor: parseUnits("0.7", 18),
+    liquidationThreshold: parseUnits("0.725", 18),
+    supplyCap: parseUnits("14000", 18),
+    borrowCap: parseUnits("7000", 18),
+    reserveFactor: parseUnits("0.25", 18),
+    protocolSeizeShare: parseUnits("0.05", 18),
+  },
+  initialSupply: {
+    amount: parseUnits("3", 18),
```

Comment:
```suggestion
    amount: parseUnits("3.208916246034338443", 18),
```

Provided here: https://uniscan.xyz/tx/0x0ba6334cba34da549a1d7ceb2f93d110ba1c4904c7650262be750ee1cf2f3588

Reply:
[eba2992](https://github.com/VenusProtocol/vips/pull/561/commits/eba2992b412b3289cceb86cc7e0f3c72e7e91840)